Makefile

```diff
@@ -181,9 +204,20 @@ clean/dist:
 # WEBCONSOLE=default make dist
 # it enables by default webconsole
 .PHONY: dist
-dist: webconsole dist/binaries
+dist: webconsole dist/binaries dist/fips
 	@echo 'Binaries generation complete. Now vcn signature is needed.'
 
+# build FIPS binary from docker image (no arm or non-linux support)
+.PHONY: dist/fips
+dist/fips: clean
+	docker build -t fips:build -f build/fips/Dockerfile.build .
+	docker run -v ${PWD}:/src -it --user root --rm fips:build -c "WEBCONSOLE=default make immudb-fips"
```

Comment:
This could potentially cause some issues regarding local user permissions, maybe we could run some docker container in the background and use `docker cp` to exchange files?

Reply:
actually this was creating problem for the builds, and if the ubuntu containers we are building off run as root, it causes an issue, hence added the `--user root` flag. I'd like to revisit this later if it ain't a blocker because the changes require checking the CI process on how it is being cloned and run as